code/datums/mutantraces.dm

```diff
@@ -836,6 +836,38 @@
 			mob.base_body_temp = initial(mob.base_body_temp)
 		. = ..()
 
+	emote(var/act, var/voluntary)
+		switch(act)
+			if ("whip")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> whips their tail", "<i>whips their tail</i>")
```

Comment:
For this and any other gender-specific entry, consider using something like [his_or_her(mob)]. It'll replace the word with the correct pronoun!

Reply:
Also the tail whip emote should check for the existence of a tail.